test/functional/p2p_ibd_txrelay.py

```diff
@@ -2,15 +2,35 @@
 # Copyright (c) 2020 The Bitcoin Core developers
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
-"""Test fee filters during and after IBD."""
+"""Test transaction relay behavior during IBD:
+- Set fee filters to MAX_MONEY
+- Don't request transactions
+- Ignore all transaction messages
+"""
 
 from decimal import Decimal
+from io import BytesIO
+import time
 
-from test_framework.messages import COIN
+from test_framework.messages import (
+        CInv,
+        COIN,
+        CTransaction,
+        msg_inv,
+        msg_tx,
+        MSG_WTX,
+)
+from test_framework.p2p import (
+        P2PDataStore,
+        P2PInterface,
+        p2p_lock
+)
 from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import hex_str_to_bytes
 
 MAX_FEE_FILTER = Decimal(9170997) / COIN
 NORMAL_FEE_FILTER = Decimal(100) / COIN
+NONPREF_PEER_TX_DELAY = 2
```

Comment:
Note this is also used in p2p_tx_download.py, can you move this in p2p.py ? If we update the production `NONPREF_PEER_TX_DELAY`, it's easier to have corresponding test constants in one place.

Reply:
Done!